src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -820,7 +816,11 @@ private void ImportProcedure(string library, ProcedureNode proc)
                             defaultArgumentNode = binaryExpr.RightNode;
                         }
                     }
-
+                    if (defaultArgumentNode != null)
+                    {
+                        var rewriter = new ElementRewriter(LibraryManagementCore.ClassTable, LibraryManagementCore.BuildStatus.LogSymbolConflictWarning);
```

Comment:
ElementRewriter is IDisposable, so would you want to dispose it after you are done?